buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -189,7 +206,33 @@ SHORT_NAMES = ['all' : ['ppc64le_linux','ppc64le_linux_xl','s390x_linux','s390x_
             'zoscm' : ['s390x_zos_cm'],
             'zosuma' : ['s390x_zos_uma'],
             'zosxl' : ['s390x_zos_xl'],
-            'zosxlcm' : ['s390x_zos_xl_cm']]
+            'zoslargeheap' : ['s390x_zos_xl'],
+            'zosxlcm' : ['s390x_zos_xl_cm'],
```

Comment:
Seems to be a non-292 change here. I guess you're just adding a missed shortname for zos.

Reply:
Yes, that is correct. Would you like it in a separate commit?

Reply:
No that's fine. It just caught my eye so I was trying to figure it out.